src/ExtractGQL.ts

```diff
@@ -193,8 +193,8 @@ export class ExtractGQL {
     return Promise.resolve().then(() => {
       const extension = ExtractGQL.getFileExtension(inputFile);
 
-      if (extension === this.extension) {
-        if (this.inJsCode) {
+      if (this.extension.indexOf(extension) > -1) {
```

Comment:
Similarly, should just be able to an array "contains" check rather than doing indexing on a string

Reply:
I assume you mean `array.includes(val)` - the reason I didn't do this is the `tsconfig.json` is set for `es6`, and `array.includes(val)` is part of the es7 lib. Do you want me to include that?